internal/store/cronjob_test.go

```diff
@@ -229,12 +256,12 @@ func TestCronJobStore(t *testing.T) {
 ` +
 				fmt.Sprintf("kube_cronjob_next_schedule_time{cronjob=\"ActiveCronJob1NoLastScheduled\",namespace=\"ns1\"} %ve+09\n",
 					float64(ActiveCronJob1NoLastScheduledNextScheduleTime.Unix())/math.Pow10(9)),
-			// TODO: Do we need to specify metricnames?
```

Comment:
I removed this TODO as it didn't need to be there.

But the question is valid: should we be consistent, we either want to always enable whitelisting or not.